internal/nodes/avi_model_l7_translator.go

```diff
@@ -297,8 +298,10 @@ func (o *AviObjectGraph) BuildPolicyPGPoolsForSNI(vsNode []*AviVsNode, tlsNode *
 
 			if path.Path != "" {
 				httpPGPath.Path = append(httpPGPath.Path, path.Path)
+				priorityLabel = host + path.Path
```

Comment:
Why is this required for EVH?

Reply:
This is for SNI (Secure).  Changes are missing for secure in SNI deployment